StRoot/StBFChain/BigFullChain.h

```diff
@@ -1698,7 +1698,8 @@ Bfc_st BFC[] = { // standard chains
    "StFcsClusterMaker","StFcsClusterMaker","Fill FCS clusters",                                     kFALSE},
   {"fcsPoint"   ,"","", "StEvent,fcsDb",
    "StFcsPointMaker","StFcsPointMaker,libMinuit","Fill FCS points",                                 kFALSE},
-
+   {"FttDat","","","StEvent","StFttRawHitMaker","StFttRawHitMaker,StEvent", 
```

Comment:
I think the 4th element should probably be "event", ensuring that StEvent exists before StFttRawHitMaker runs in the chain.

Reply:
The `"StEvent"` option also turns on `detDb`:
```
{"StEvent"   ,"","","globT,SCL,TRGDef,StBichsel,EmcUtil,TbUtil,detDb","","StEvent","Load StEvent",kFALSE},
```
Perhaps it didn't do so sometime in the past, but now it does. Regardless, `"Event"` ensures that there will be an StEvent for you to fill, not just that StEvent libraries are loaded/available at run-time.

Reply:
Ok I see, I am fine with that then. Thanks

Reply:
Hi Gene,

I just had a chance to test this. In fact I find that the test chain that I provided:
```
root4star -b -q -l 'bfc.C( 5, "in, db, StEvent, FttDat", "input.daq" )'
```
runs fine if FttDat uses "StEvent" as the 4th item. It also saves the data into an StEvent object, no problem.  
However, if I change it to "Event" or "MakeEvent" then the inclusion of "StTpcDb" leads to an error

```
root4star: .sl73_gcc485/obj/StRoot/StTpcDb/StTpcDb.cxx:276: float StTpcDb::DriftVelocity(Int_t): Assertion `mUc > 0' failed.
Abort
```

Is there a simple way to solve this? Even if this is not an issue for production chains, I still need a minimal chain that I can run for development purposes